springfield/cms/migrations/0018_alter_freeformpage_content_and_more.py

```diff
@@ -0,0 +1,28 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Generated by Django 5.2.7 on 2025-10-27 21:29
+
+from django.db import migrations
+
+import springfield.cms.fields
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("cms", "0017_merge_20251023_0634"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="freeformpage",
+            name="content",
+            field=springfield.cms.fields.StreamField(block_lookup={}),
+        ),
+        migrations.AlterField(
+            model_name="whatsnewpage",
+            name="content",
+            field=springfield.cms.fields.StreamField(block_lookup={}),
```

Comment:
This all looks really promising, thanks! 

Just to check I'm on the same page: these two models have new migrations because this is the first time we're skipping the StreamField's content, yep?